src/app/public/modules/lookup/lookup.component.ts

```diff
@@ -113,6 +121,33 @@ export class SkyLookupComponent
   @Input()
   public idProperty: string;
 
+  /**
+   * Specifies the selection mode that determines whether users can select one item or
+   * multiple items. The valid options are `single` and `multiple`.
```

Comment:
```suggestion
   * 
```
We don't list the valid options in the property descriptions when the reference docs include a link to another grid with those options listed.

Reply:
Done